settings.example.json

```diff
@@ -34,6 +35,8 @@
       "req_per_sec": 130,
       "burst": 1
     },
+    "project_api_key": "ipfs-project-id",
```

Comment:
Since IPFS reader and write urls can be different, IMO it would be better to have separate keys and secrets for both.

Reply:
Makes sense, I have added separate authentication config for reader and writer client